metaflow/util.py

```diff
@@ -50,19 +50,14 @@ def unquote_bytes(x):
 
     from shlex import quote as _quote
 
-if sys.version_info >= (3, 7):
```

Comment:
what's the context for this change?

Reply:
so I could use `NamedTuple` and have the proper types.